app/packages/looker-3d/src/fo3d/AssetErrorBoundary.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+export class AssetErrorBoundary extends React.Component<any> {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError(error) {
+    console.error(error);
+    return { hasError: true };
+  }
+
+  render() {
+    // @ts-ignore
+    if (this.state.hasError) {
+      // todo: add indicator in canvas that asset failed loading
+      return null;
```

Comment:
Can in indicator / informative message be added? Is separate `AssetErrorBoundary` required?